libs/eavmlib/src/network.erl

```diff
@@ -62,6 +62,11 @@
 -type sta_config() :: {sta, [sta_config_property()]}.
 
 -type mac() :: binary().
+-type ap_channel_cfg() :: {ap_channel, 1..14}.
+% This setting is only used while a devise is operation is AP mode only. If `ap_channel_cfg' is
+% set, it will be temporarily changed to match the associated access point if ap+sta is used and
+% the station is associated with an access point. This is a hardware limitation due to the modem
+% only being able to operate on a single channel at a time.
```

Comment:
That makes sense, this effect is likely to go unnoticed, and does not impact user applications. This would be better information to include in the Network Programming Guide. It should be included somewhere in the documentation since, at least on the esp32 platform, this does trigger a low-level channel change event that can appear in the console log.